guides/advanced_guides/relevancy.md

```diff
@@ -0,0 +1,204 @@
+
+MeiliSearch proposes default ranking rules and has given them a default order as well. **This order can be modified, rules can be deleted and new ones can be added.**
+
+[For some in-depth explanations about the mechanism and about each default sorting rule](https://github.com/meilisearch/MeiliSearch/issues/358).
+
+To be able to play with these rules and match them to the needs of your dataset, it is important to understand how each works and how to create new ones.
+
+## Ranking rules
+
+MeiliSearch has built-in ranking rules. These rules are essential to the relevance of the search engine.
+
+Each of the rules has a role in finding the right documents for the given search query.
+The order in which the rules are set in the settings affects the importance of the rule. The first rule is the most important, then the second and so on. By default, Meilisearch has these rules in a specific order, thinking it meets the most standard needs. This order can be changed in the settings to fit your needs.
+
+Using a [bucket sort](/guides/advanced_guides/bucket_sort.md) algorithm, MeiliSearch uses rule by rule to find documents, until the desired amount of documents is obtained.
+
+Here is the list of all the rules that are executed in this specific order by default:
+
+#### 1. Typo
+the `_typo` rule sorts by ascending numbers of typos.
+
+This means that a matched word with fewer typos is more relevant than a matched word with more typos.
+
+#### 2. Words
+The `_words` rule sort by decreasing numbers of matched query words.
+
+#### 3. Proximity
+The `_proximity` rule sort by increasing proximity of query words in hits.
+
+#### 4. Attribute
+
+The `_attribute` rule sorts by ascending [attribute importance](/guides/advanced_guides/relevancy.md#attributes-importance).
+
+#### 5. Words position
+The `_words_position` sort according to the position of query words in the attribute. The start is better than the end.
+
+#### 6. Exact
+The `_exact` - Sort by the similarity of the matched words with the query words. Words that are exactly the same are better than prefixes.
+
+#### Examples
+
+:::: tabs
+
+::: tab Typo
+
+![Image from alias](../../public/image/vogli3.png)
+
+### Typo
+
+- `vogli` : 0 typo
+- `volli` : 1 typo
+
+The typo sorts the results by ascending typos on matched query words.
+
+Because `typo` is set as the first rule of our ranking rules, the number of `typos` sorts the document before
+
+:::
+
+::: tab Words
+![Image from alias](../../public/image/new_house.png)
+
+### Words
+
+Matched words :
+- **The Housemaid**: `Housemaid`, `new`, `house`, `housemaid`, `housework`: 5 words.
+- **The open House**: `House`, `new`, `house`: 3 words.
+
+The `words` ranking rule sorts the results by descending number of matching query words.
```

Comment:
```suggestion
The `words` rule sorts the results by descending number of matching query words.
```

Reply:
(I removed it because you don't use in the same kind of sentences you did. But grammatically and technically it's right of course)